pkg/task/meta_test.go

```diff
@@ -210,7 +215,7 @@ queries:
 
 	for name, mock := range tests {
 		t.Run(name, func(t *testing.T) {
-			mte, err := newMetaTaskExecutor(mock.id, mock.yaml, mock.values)
+			mte, err := newMetaTaskExecutor(mock.yaml, mock.values)
 			if err != nil && !mock.isErr {
```

Comment:
it seems mock.isErr is not checked with obtained err.

Reply:
isErr is a bool. If isErr is false i.e. not expected & there is an actual error then the test case fails.
The test case does not deal with checking the error message etc.